tools/property-extractor/property_extractor.py

```
@@ -0,0 +1,230 @@
+        if any(path in implementation_value for path in file_paths):
+            if len(properties) > 0:
+                files_with_properties.append((fp, properties))
+
+    
+
+    return files_with_properties
+
+
+def transform_files_with_properties(files_with_properties):
+    transformers = [
+        TypeTransformer(),
```

Ruff (0.8.2)

`TypeTransformer` may be undefined, or defined from star imports

(F405)